addons/account/models/account_move.py

```diff
@@ -1784,6 +1786,11 @@ def _compute_abnormal_warnings(self):
                 wiggle=move.currency_id.format(wiggle_room_amount),
             )
 
+    @api.depends('line_ids.tax_ids')
+    def _compute_taxes_legal_notes(self):
+        for move in self:
+            move.taxes_legal_notes = ''.join(tax.invoice_legal_notes for tax in set(move.line_ids.tax_ids) if not is_html_empty(tax.invoice_legal_notes))
```

Comment:
Use `odoo.tools.OrderedSet` or `odoo.tools.unique` to be able to keep a deterministic order
```suggestion
            move.taxes_legal_notes = ''.join(
                tax.invoice_legal_notes 
                for tax in OrderedSet(move.line_ids.tax_ids) 
                if not is_html_empty(tax.invoice_legal_notes)
            )
```

Reply:
Thanks for the review William. I've made the change suggested. I've discussed your idea with Laura but right now she thinks it's only necessary on the pdf of invoices. Also, we decided it is better to use Text fields instead of HTML fields.

Reply:
> we decided it is better to use Text fields instead of HTML fields

Why?
HTML seems a lot better without more context

Reply:
> > we decided it is better to use Text fields instead of HTML fields
> 
> Why? HTML seems a lot better without more context

Because in most cases, users will only need to write a few words. HTML fields enable more advanced formatting and styling that does not seem useful in this case.

Reply:
But is there any inconvenient to use a HTML field?
It seems that at worst, people won't use it.
At best, people use it.

But it shouldn't have any real cost.

Reply:
I think you're right, they can simply write text if they want, and at least they have the possibility for further customization if they want with an html field.